### DISCUSSION TITLE
About es8.x and es7.x

### DISCUSSION THREAD

**Discussion by Zhang21** [jertel/elastalert2#1452] [Q&A] [answered]
elasticsearch-py in requirements.txt

```
elasticsearch==7.10.1
```

my es cluster: 8.13

From pypi, the [elasticsearch-7.10.1](https://pypi.org/project/elasticsearch/7.10.1/) does not support es 8.x，so elastalert2 doesn't work, it doesn't hit rules, and return `WARNING:elasticsearch:DELETE http://xxx:9200/_search/scroll [status:404 request:0.001s]` error logs。

**Comment by Zhang21** [comment]
What can I do to make elastalert2 fit my es cluster 8.13?

**Comment by jertel** [comment, marked as answer]
ElastAlert 2 does indeed support Elastic 8.x. There is a specific reason the Python Elastic lib is pinned at 7.10, and if you search these discussions and the PR and issue history you can find more information about it.

The WARNING you pasted above is a non-fatal warning only. It does not impact the operation of ElastAlert 2.

**Reply by Zhang21** [reply]
Got it.
